samples/terminal-plugin/package.json

```diff
@@ -0,0 +1,33 @@
+{
+      "name": "terminal-plugin",
+      "publisher": "theia",
+      "keywords": [
+        "theia-plugin"
+      ],
+      "version": "0.0.1",
+      "license": "EPL-2.0",
+      "files": [
+        "src"
+      ],
+      "devDependencies": {
+        "@theia/plugin": "next",
+        "@theia/plugin-packager": "latest",
+        "rimraf": "2.6.2",
+        "typescript-formatter": "7.2.2",
+        "typescript": "2.9.2"
```

Comment:
3.1.3 ?

Reply:
The same way we are using typescript-3.1.3 for Theia 0.3.19

Reply:
I don't mind newer version but for this sample 2.9.2 is enough for compilation. Most our samples uses this version. For me newer version make sense when we used new language syntax features.

Reply:
for theia extensions it's kind of mandatory as we'll compile extensions with typescript version of theia
but for plug-ins and examples, yes, as it only transpiles code when we want, it can be any typescript version

Reply:
Ok. Just a purpose to use the same version